here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -476,25 +482,103 @@ paths:
           $ref: '#/components/responses/ErrorResponse504'
         '513':
           $ref: '#/components/responses/ErrorResponse513'
+    put:
+      tags:
+        - Write Features
+      summary: Upsert features in the space
+      description: >
+        This method allows to upsert features in the storage associated (directly or via event handler) with the space.
+        If a feature does not exist yet, it is created, if it is already available, it will be updated.
+        For each existing feature, if UUID is specified and matches the current UUID stored in Naksha, the feature will be updated.
```

Comment:
Suggest rephrasing this statement to also indicate behaviour for "missing uuid"... may be like this:
`For each existing feature, if UUID is not provided OR if UUID provided matches with what is stored in Naksha, the feature will be updated. Recommendation is to always provide correct UUID to avoid inconsistency due to concurrency.`
Also UUID is a string located at "properties -> @ns:com:here:xyz -> uuid"